### PR TITLE
Advanced trading/market mode

### PR DESCRIPTION
**Pull request by dkildar**

### CODE REVIEW

File: src/common/components/market-advanced-mode/stake-widget-header-options.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import React, { ChangeEvent, useState } from "react";
+import { _t } from "../../i18n";
+import { Button } from "react-bootstrap";
+import { allStakeSvg, buyStakeSvg, sellStakeSvg } from "../../img/svg";
+
+export enum StakeWidgetViewType {
+  All = "all",
+  Sell = "sell",
+  Buy = "Buy"
```
Comment (feruzm) [inline src/common/components/market-advanced-mode/stake-widget-header-options.tsx:9]: buy capitalized, on purpose?

File: tsconfig.json [hunk 2]
```diff
@@ -42,12 +35,8 @@
     "jest",
     "razzle.config.js",
     "healthCheck.js",
-    "src/desktop/**/*.js"
+    "src/desktop/**/*.js",
+    "node_modules/react-use"
```
Comment (feruzm) [inline tsconfig.json:39]: Why do we need to add this here?
I suspect warning in console, coming from here...
Screenshot 2023-01-14 at 07 30 23

Comment (dkildar) [reply tsconfig.json:39]: Not from here. Warning because we have too old typescript version

Comment (dkildar) [reply tsconfig.json:39]: I tried to fix it but it didn't help. Will return as it was

File: src/common/components/market-advanced-mode/stake-widget-header-options.tsx (hunk 1 above)
Comment (dkildar) [reply src/common/components/market-advanced-mode/stake-widget-header-options.tsx]: Fixed

**Review by feruzm — APPROVED** [review]
🎉🛫

**Pull request merged**